packages/preview-middleware-client/types/global.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module 'sap/open/ux/preview/global' {
```

Comment:
[Bootstrap.ts](https://github.com/SAP/open-ux-tools/pull/2192/files#diff-8d76ca9a519d1d51f8dcbd0428e926d6ce2e33cfc4120c47390fbd9d2062bc52) has its own interface definition for `Window`. In my opinion we should either use a global one or a local one but not both in parallel.

Reply:
Absolutely, one is enough. And I also don't understand the wrapping 'declare' at the top, it is not necessary. I'd keep it global (and not locally in the bootsrap.ts) because it is globally valid in our context. We might even enhance it further with other global configs in UI5 (or QUnit).

Reply:
@heimwege please check again